packages/rocketchat-contacts/server/index.js

```diff
@@ -94,6 +96,58 @@ Meteor.methods({
 		}
 		return provider.queryContacts(weakHashes);
 	},
+	getInviteLink() {
+		const user = Meteor.user();
+		if (!user) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'getInviteLink',
+			});
+		}
+		if (!RocketChat.settings.get('Contacts_Dynamic_Link_APIKey')) {
+			throw new Meteor.Error('error-invalid-config', 'Contacts_Dynamic_Link_APIKey not configured', {
+				method: 'getInviteLink',
+			});
+		}
+
+		if (!RocketChat.settings.get('Contacts_Dynamic_Link_DomainURIPrefix')) {
+			throw new Meteor.Error('error-invalid-config', 'Contacts_Dynamic_Link_DomainURIPrefix not configured', {
+				method: 'getInviteLink',
+			});
+		}
+
+		if (!RocketChat.settings.get('Contacts_Dynamic_Link_AndroidPackageName')) {
+			throw new Meteor.Error('error-invalid-config', 'Contacts_Dynamic_Link_AndroidPackageName not configured', {
+				method: 'getInviteLink',
+			});
+		}
+
+		const server = RocketChat.settings.get('Site_Url');
+
+		this.unblock();
+		try {
+			const result = HTTP.call('POST', `https://firebasedynamiclinks.googleapis.com/v1/shortLinks?key=${ RocketChat.settings.get('Contacts_Dynamic_Link_APIKey') }`, {
+				data: {
+					dynamicLinkInfo:{
+						domainUriPrefix: RocketChat.settings.get('Contacts_Dynamic_Link_DomainURIPrefix'),
+						link: `${ server }direct/${ user.username }`,
```

Comment:
Missing a '/' here, between {server} and 'direct' :  link: `${ server }direct/${ user.username }`,